src/ansys/fluent/core/services/datamodel_se.py

```diff
@@ -867,14 +827,40 @@ def __init__(
         service: DatamodelService,
         rules: str,
         path: Path,
-        static_info,
+        arg,
     ):
         PyCommandArgumentsSubItem.__init__(
-            self, parent, attr, service, rules, path, static_info
+            self, parent, attr, service, rules, path, arg
         )
         PyParameter.__init__(self, service, rules, path)
 
 
+class AccessorModes(Enum):
+    """Provides the standard Fluent launch modes."""
+
+    # Tuple:   Name, Solver object type, Meshing flag, Launcher options
+    TEXT = (["String", "ListString", "String List"], PyTextualCommandArgumentsSubItem)
+    NUMBER = (
+        ["Real", "Int", "ListReal", "Real List", "Integer"],
```

Comment:
@prmukherj Is it correct to include the numerical list types here. The reason for the Numerical class is for min and max which only apply to the single valued numerical types.

Reply:
@prmukherj are the tests failures there simply because we are waiting for the Fluent docker update?

Reply:
Yes, the test failures are due to the pending update in the server side. Regarding your previous comment, we can keep numerical list as PyParamter only.